docs/openshift/kctlr-use-as3-backend.rst

```diff
@@ -0,0 +1,89 @@
+:product: BIG-IP Controller for Kubernetes
+:type: task
+
+.. _use_as3_backend:
+
+Use AS3 as Orchestration Backend
+================================
+
+This document provides step-by-step instructions for using `**AS3** <http://example.com>` as orchestration back end. An
+orchestration backend is a component of the controller that creates objects in the BIG-IP.
+
+Requirements
+------------
+1. BIG-IP Controller for Openshift 1.10 or higher
+2. BIG-IP version 12.1.x or later.
+3. AS3 3.11 or higher.
+4. A BIG-IP user account with **Administrator** role.
+
+Installation and Configuration
+------------------------------
+Before operating the controller with AS3 orchestration backend, you need to install AS3 package in BIG-IP as in the
+`AS3 Installation Guide <https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/userguide/installation.html>`.
+
+Once AS3 package is installed in BIG-IP, restart the controller with the following command line parameter
+`--agent=as3`. A sample controller deployment is provided below:
+
+.. code-block:: YAML
+   :emphasize-lines: 9
+   args: [
+         "--bigip-username=$(BIGIP_USERNAME)",
+         "--bigip-password=$(BIGIP_PASSWORD)",
+         "--bigip-url=10.10.10.10",
+         "--bigip-partition=myPartition",
+         "--pool-member-type=cluster",
+         "--openshift-sdn-name=/Common/openshift_vxlan",
+         "--manage-routes=true",
+         "--agent=as3"
+         ]
+
++---------------------+---------------+---------+--------------------------------------------------------------+
+| Argument            | Values        | Default |                      Description                             |
++=====================+===============+=========+==============================================================+
+| --agent             | as3, cccl     | cccl    | If as3 is given CIS creates one additional partition with    |
+|                     |               |         | name <partition>_AS3 where <partition> is provided through   |
+|                     |               |         | argument --bigip-paritition                                  |
++---------------------+---------------+---------+--------------------------------------------------------------+
+
```

Comment:
May be we need to some info on --manage-ingress option too. May not be in this doc.

Reply:
That is out of the scope of this doc.